scripts/kernel_upgrade.pl

```diff
@@ -288,7 +288,7 @@ ()
 
 foreach my $name (@$list) {
 	next if($git->ls($start, "$name$to"));
-	my $new=$git->ls($start, "$name$from");
+	my $new=$git->ls('', "\"$name$from\"");
```

Comment:
Then when looking some more, realize the quotes should instead be added in `GitImporter->ls()`.